### ISSUE TITLE
Task: Testing

### ISSUE THREAD

**Issue by carolinan** [WordPress/twentytwentyone#305]
The theme needs to be tested on WordPress 5.3 and newer.

Older version of WP can be installed using **wpenv** which is already available to install with this theme (with npm and Docker).
### Before starting wpenv, the version must be changed in the file `.wp-env.override.json`
https://developer.wordpress.org/block-editor/packages/packages-env/#wp-env-override-json
https://developer.wordpress.org/block-editor/packages/packages-env/#a-complete-testing-environment

To install 5.3, you would add:
`    "core": "WordPress/WordPress#5.3.0",`

- [x] 5.3
- [x] 5.4
- [x] 5.5
- [ ] 5.6 beta-rc
(Gutenberg 9.1 -9.2?)

-----

When testing on an older version, please open new issues for problems that are not already reported, and use the 
issue template to include version number, device and browser.

I you have successfully completed testing, please leave a comment on this issue.
WordPress version number, device, browser, with or without Gutenberg.

**Comment by carolinan** [comment]
5.3, 5.4 tested and low priority issues opened. The main problem found was the overlapping style for the columns block.

**Comment by carolinan** [comment]
I don't think we need a reminder to test on 5.6 beta-rc since that is what we are at now. With testing on 5.3-5.5 done to the best of our ability, I will close this.
